paimon-common/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -438,7 +438,7 @@ public class CoreOptions implements Serializable {
     public static final ConfigOption<MemorySize> TARGET_FILE_SIZE =
             key("target-file-size")
                     .memoryType()
-                    .defaultValue(MemorySize.ofMebiBytes(128))
+                    .defaultValue(MemorySize.ofMebiBytes(256))
```

Comment:
Maybe we can set primary table to 128MB, but set append table to 256MB

Reply:
done